### PR TITLE
Updating logging in PatchManager

### PR DESCRIPTION
**Pull request by CoreyHendrey**
I'm starting to update the logging in Patch Manager.
Is there anything, in particular, you would like to see logged?

I noticed there is already quite a bit of logging in there.

I've added a few things but I am not sure they will be very useful at all.

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
I'd like to see these things logged:
- When a spawn type is removed log the name
- When a spawn density is changed, log the name and new density
- When a species doesn't spawn because no population, log the name

This way it is possible to follow along and see what spawn types are active and if they have their densities correctly updated.

**Comment by CoreyHendrey** [comment]
Ok. this seems to fulfill all those requirements.

File: src/microbe_stage/PatchManager.cs [hunk 1]
```diff
@@ -213,7 +233,12 @@ private void RemoveNonMarkedSpawners()
 
     private void ClearUnmarkedSingle(List<CreatedSpawner> spawners)
     {
-        spawners.RemoveAll((item) => item.Marked == false);
+        spawners.RemoveAll((item) =>
+        {
+            GD.Print("Removed ", item.Name, " spawner.");
```
Comment (hhyyrylainen) [inline src/microbe_stage/PatchManager.cs:240]: Doesn't this always end up printing?

File: src/microbe_stage/PatchManager.cs [hunk 2]
```diff
@@ -193,6 +213,8 @@ private void UpdateLight(Biome biome)
 
     private void UnmarkAllSpawners()
     {
+        GD.Print("Umarking all spawners.");
```
Comment (hhyyrylainen) [inline src/microbe_stage/PatchManager.cs:216]: Forgot to include in the review that this print doesn't add anything and should be removed.

File: src/microbe_stage/PatchManager.cs (hunk 1 above)
Comment (CoreyHendrey) [reply src/microbe_stage/PatchManager.cs]: Yep! Sorry. Fixed.

File: src/microbe_stage/PatchManager.cs (hunk 2 above)
Comment (CoreyHendrey) [reply src/microbe_stage/PatchManager.cs]: Fixed.

**Pull request merged**